wikiconv/conversation_reconstruction/dataflow_test.py

```diff
@@ -14,12 +15,35 @@
 from apache_beam.testing import test_pipeline
 from apache_beam.testing import util
 from wikiconv.conversation_reconstruction import dataflow_main
+import six
 
 
 class FakeStorageClient(object):
   pass
 
 
+def ordered(obj):
+  if isinstance(obj, dict):
+    return sorted((k, ordered(v)) for k, v in obj.items())
+  if isinstance(obj, list):
+    return sorted(ordered(str(x)) for x in obj)
```

Comment:
This converts list elements to strings? What if the list contains more complex data, like dicts or other lists?

Why is `ordered()` called on` str(x)`, isn't that always a no-op?

Reply:
Just need an order, but the lists can be heterogenous, so I used a key fn instead.

Reply:
but isn't the `ordered` a no-op, because `str` always returns a string, which isn't a dict or a list, so the call to `ordered` will always return the original string again?

Reply:
oops missed your latest change